### DISCUSSION TITLE
ViolentMonkey 2.13.5可能存在问题，导致无法加载脚本

### DISCUSSION THREAD

**Discussion by Kosette** [the1812/Bilibili-Evolved#3920] [综合讨论 / General]
Firefox版本的暴力猴插件自动更新到了2.13.5，结果脚本就无法正常加载了，与此同时Edge版本的更新还在2.13.4，一切都正常工作。

看了一下暴力猴的仓库，已经发布了2.13.6更新，修复了一些和加载有关的[BUG](https://github.com/violentmonkey/violentmonkey/releases/tag/v2.13.6)，浏览器商店尚未审核通过，不知道是不是解决了这个问题，有心人可以从仓库手动安装暴力猴2.13.6版本测试结果。

**Comment by Geeyun-JY3** [comment]
有相关反馈：https://github.com/the1812/Bilibili-Evolved/issues/3918#issuecomment-1374141914
